js/templates/modals/listingDetail/listing.html

```diff
@@ -175,7 +175,7 @@ <h2 class="flexExpand js-cryptoTitle cryptoTitle"></h2>
             </div>
           <% } else if (ob.metadata.contractType === 'PHYSICAL_GOOD') { %>
             <div>
-              <%= ob.polyT('listingDetail.condition', { condition: `<b>${ob.polyT(`conditionTypes.${ob.item.condition.toUpperCase()}`)}</b>` }) %>
```

Comment:
This could be done natively using the polyglot function by passing in a `_` option, e.g:

```
 <%= ob.polyT('listingDetail.condition', { condition: `<b>${ob.polyT(`conditionTypes.${ob.item.condition.toUpperCase()}`, { _: ob.item.condition })}</b>` }) %>
```

Reply:
Thanks! I completely forgot about the fallback parameter.